### PR TITLE
Example of sending user to manage account

### PR DESCRIPTION
**Pull request by kumar303**
Supports https://github.com/mozilla/payments-ui/issues/202

### CODE REVIEW

File: public/css/index.css [hunk 1]
```diff
@@ -96,6 +96,11 @@ button#signin {
     padding: 0.3em;
 }
 
+button#signin, button#manage {
```
Comment (muffinresearch) [inline public/css/index.css]: Could you just use the ids sans "button"?

**Comment by muffinresearch** [comment]
Would be nice to add the necessary bits so this will point at the webpack enabled version too. See https://github.com/mozilla/payments-example/pull/23/files#diff-e70fdf29cba0fbb5a52462aed7e5c97dR14

**Comment by muffinresearch** [comment]
r+wc

File: public/index.html [hunk 2]
```diff
@@ -31,15 +31,17 @@
           </ul>
         </ul>
       </div>
+      <!--
```
Comment (muffinresearch) [inline public/index.html:34]: Could just kill this block?

**Comment by kumar303** [comment]
good call on hot reloading. Filed: https://github.com/mozilla/payments-example/issues/24

File: public/index.html (hunk 2 above)
Comment (kumar303) [inline public/index.html:34]: The only reason I left it is because once the FxA storage server is up then we'll probably add this back in.

**Pull request merged**